docs/data/joy/getting-started/templates/TemplateCollection.js

```diff
@@ -10,10 +10,10 @@ import Link from '@mui/joy/Link';
 import List from '@mui/joy/List';
 import Button from '@mui/joy/Button';
 import Typography from '@mui/joy/Typography';
-import SvgIcon from '@mui/joy/SvgIcon';
 import Visibility from '@mui/icons-material/Visibility';
 import CodeRoundedIcon from '@mui/icons-material/CodeRounded';
-import codeSandbox from 'docs/src/modules/sandbox/CodeSandbox';
+import Edit from '@mui/icons-material/Edit';
```

Comment:
I think a different icon would be better suited for this. The pencil makes me think it would start editing inline (like how GitHub works)

What do you think of `Terminal`, `Code`, `Biotech`, `PrecisionManufacturing`, `Construction`, `DisplaySettings`, or `OpenInBrowser`